.gitignore

```
@@ -1,4 +1,12 @@
 *.pyc
+new_tests/__pycache__
+new_tests/.pytest_cache
+new_tests/output/2023MQ5.xml
+new_tests/output/read_xml_out.txt
+new_tests/output/K20Q04A_test.xml
+new_tests/output/trksub_sub.obs
+new_tests/output/obs.psv
```

I think this can be replaced with just `new_tests/output` to exclude the whole output directory.
```suggestion
new_tests/output
```

---

I've pushed a commit making this change per suggestion from @stevechesley